x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -94,6 +94,16 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.allow_cloud_features',
+    first_supported_version: '8.18',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.allow_cloud_features',
+      {
+        defaultMessage: `Advanced option to selectively choose which external services are allowed, valid keywords are "sample-collection,reputation-lookup,malware-lookup,artifacts-update,staged-artifacts-rollout". Everything is allowed by default. To disallow all a special keyword "none" can be used. The option imposes severe limitation on Endpoint functionality. It's meant only for telemetry extra-avoidant users, especially for FedRAMP High users.`,
```

Comment:
```suggestion
        defaultMessage: `Advanced option to selectively choose which external services are allowed, valid keywords are "sample-collection,reputation-lookup,malware-lookup,artifacts-update,staged-artifacts-rollout". Everything is allowed by default, but if any comma separated value(s) are provided all other features are disabled. To disallow all a special keyword "none" can be used. The option imposes severe limitation on Defend functionality. It's meant only for telemetry extra-avoidant users users.`,
```

Reply:
`Advanced option to selectively choose which external services are allowed. Everything is allowed by default, but if any comma separated value(s) are provided all other features are disabled. To disallow all a special keyword "none" can be used. The option imposes severe limitation on Defend functionality. It's meant only for telemetry extra-avoidant users users.
8.18 valid keywords are: sample-collection,reputation-lookup,malware-lookup,artifacts-update,staged-artifacts-rollout`

I'm not sure how to convey this, but additional cloud features could be added in the future

Reply:
Perhaps 


> Advanced option to selectively choose which external services are allowed. Everything is allowed by default, but if any comma separated value(s) are provided all other features are disabled. This means future cloud features will be disabled unless explicitly enabled when this advanced option is in use. To disable all cloud features use the special keyword "none". This option imposes severe limitation on Defend functionality. It's meant only for telemetry extra-avoidant users. As of Elastic Agent 8.18 the valid keywords are: sample-collection, reputation-lookup, malware-lookup, artifacts-update, staged-artifacts-rollout. Default: all cloud features are enabled.